modules/internal/ChapelArray.chpl

```diff
@@ -1613,12 +1613,14 @@ module ChapelArray {
       return this((...r));
     }
 
-    pragma "no doc"
+    /* Return an array slice halting if the range contains elements
+       that are not local. */
```

Comment:
This routine doesn't take a range, it takes a vararg number of ranges (essentially, an anonymous rectangular domain).